ibm_catalog.json

```diff
@@ -107,13 +108,12 @@
                 ]
               },
               {
-                "key": "prefix",
-                "required": true,
-                "description": "Prefix to add to all resources created by this solution. To not use any prefix value, you can enter the string `__NULL__`."
```

Comment:
Oh you should add this back - this is one of the exceptions we want to force

Reply:
Actually we need this description too as it is specific to Schematics / Projects. It will override the terraform based descrption